conans/client/graph/graph_binaries.py

```diff
@@ -219,26 +220,27 @@ def _evaluate_node(self, node, build_mode, update, remotes):
         node.binary_remote = remote
 
     @staticmethod
-    def _compute_package_id(node, default_package_id_mode):
+    def _compute_package_id(node, default_package_id_mode, process_options=True):
```

Comment:
Add a comment about the `process_options` something like: Do not process again the options when the package id is being recalculated (previously unknown)

Reply:
Instead of documented this weird argument, I have better separated the responsibilities, now 2 functions clearly show what they do, and when they are called.